### PR TITLE
docs: added architecture section

### PR DESCRIPTION
**Pull request by maocorte**
* added a minimal architecture section inside documentation
* changed files/menu hierarchy using `sidebar_position` instead of file names

### CODE REVIEW

File: docs/docs/l7_architecture.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+In this section we'll explore the architecture of the Radicalbit AI platform.
+The image below show all the components of the platform:
+
+![Alt text](/img/architecture/architecture.png "Architecture")
+
+## API
+
+API is the core of the platform, it exposes all the functionalities via REST APIs.
+It require a PostgreSQL database to store data and a Kubernetes cluster to run Spark jobs for metrics evaluations.
```
Comment (mmariniello90) [inline docs/docs/l7_architecture.md:15]: @maocorte there's a typo.
Errata: It require;
Corrig: It requires

File: docs/docs/l7_architecture.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 1
+---
+
+# Architecture
+
+In this section we'll explore the architecture of the Radicalbit AI platform.
+The image below show all the components of the platform:
```
Comment (mmariniello90) [inline docs/docs/l7_architecture.md:8]: @maocorte there's a typo.
Errata: The image below  show;
Corrige: The image below  shows

File: docs/docs/l7_architecture.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+![Alt text](/img/architecture/architecture.png "Architecture")
+
+## API
+
+API is the core of the platform, it exposes all the functionalities via REST APIs.
+It require a PostgreSQL database to store data and a Kubernetes cluster to run Spark jobs for metrics evaluations.
+To store all dataset files a distributes storage is used.
```
Comment (mmariniello90) [inline docs/docs/l7_architecture.md:16]: @maocorte there's a typo.
Errata: distributes;
Corrige: distributed

File: docs/docs/l7_architecture.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+REST APIs could be used via user interface or using the provided Python SDK.
+
+## UI
+
+To use REST APIs with a human friendly interface, a UI is provided.
+It covers all the implemented APIs, starting from models creation ending with all metrics visualization.
```
Comment (mmariniello90) [inline docs/docs/l7_architecture.md:22]: @maocorte
I suggest adding an "and": " starting from models creation AND ending with all metrics visualization"

File: docs/docs/l7_architecture.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## SDK
+
+To interact with API programmatically, a Python SDK is provided.
+SDK implements all functionalities exposed via REST API.
```
Comment (mmariniello90) [inline docs/docs/l7_architecture.md:27]: @maocorte
I suggest adding a "The" at the beginning of the sentence: "The SDK implements all..."

File: docs/docs/architecture.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 6
+---
+
+# Architecture
+
+In this section we'll explore the architecture of the Radicalbit AI platform.
```
Comment (aleconf) [inline docs/docs/architecture.md:7]: @maocorte Remove «'ll».

File: docs/docs/architecture.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+API is the core of the platform, it exposes all the functionalities via REST APIs.
+It requires a PostgreSQL database to store data and a Kubernetes cluster to run Spark jobs for metrics evaluations.
+To store all dataset files a distributed storage is used.
+REST APIs could be used via user interface or using the provided Python SDK.
+
+## UI
+
+To use REST APIs with a human friendly interface, a UI is provided.
+It covers all the implemented APIs, starting from models creation and ending with all metrics visualization.
```
Comment (aleconf) [inline docs/docs/architecture.md:22]: @maocorte «model creation», not «models creation».

File: docs/docs/architecture.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+## SDK
+
+To interact with API programmatically, a Python SDK is provided.
```
Comment (aleconf) [inline docs/docs/architecture.md:26]: @maocorte Write *Python SDK* in italics to show it is an entry in the documentation to keep it consistent with other files.

**Review by aleconf — CHANGES_REQUESTED** [review]
Asked for three changes in the `docs/docs/architecture.md` file. Save for that it looks good to me.

**Review by aleconf — APPROVED** [review]

**Pull request merged**